recipes/libavif/install-tests.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 83f0214..dbcb7cf 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -510,6 +510,7 @@ if(AVIF_BUILD_TESTS)
+         tests/compare.c
+         tests/testcase.c
+     )
++    install(TARGETS aviftest)
```

Comment:
Is this installing the `aviftest` in the final prefix? Is that desired?

Reply:
I did this because avitftest wouldn't link to the shared library when I didn't install it as part of the build step and listed it in the test/source_files section. The test would just fail complaining that it couldn't find libavif.so.

I'm open to other solutions.

Reply:
Have you ever used `conda-inspect linkages`? Is this tool capable of correcting broken links to libraries in a package? If so, we could use it instead of this patch.

Reply:
I think what would be more ideal woudl be to run the tests _inside_ the build script (`build.sh`) (as they would be normally run). That would still make the recipe fail should there be an issue. 

In the test section the thing you have now is really good!

Reply:
Because this way, every user will get the aviftest binary shipped which they probably don't really want or need :)

Reply:
Yep. Makes sense.